fabric-registry-sync-v0/src/main/java/net/fabricmc/fabric/impl/registry/sync/FabricRegistryInit.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.registry.sync;
+
+import net.minecraft.util.registry.Registry;
+
+import net.fabricmc.api.ModInitializer;
+import net.fabricmc.fabric.api.event.registry.RegistryAttribute;
+import net.fabricmc.fabric.api.event.registry.RegistryAttributeHolder;
+
+public class FabricRegistryInit implements ModInitializer {
+	@Override
+	public void onInitialize() {
+		// Synced in PlaySoundS2CPacket.
+		RegistryAttributeHolder.get(Registry.SOUND_EVENT)
```

Comment:
Wondering if we can sort these calls by their order of appearance in `Registry` class or how are we going to organize and make sure no vanilla registry is missed?

Reply:
They should be in the right order, making sure they are all covered should be quite easy. Possibly just printing out that a registry hasnt been accounted for?